### PR TITLE
Compound documents are not returning links #125

### PR DESCRIPTION
**Pull request by blabno**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/agco/harvesterjs/pull/126%23discussion_r36112934%22%2C%20%22https%3A//github.com/agco/harvesterjs/pull/126%23discussion_r36113913%22%2C%20%22https%3A//github.com/agco/harvesterjs/pull/126%23issuecomment-127401270%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/agco/harvesterjs/pull/126%23issuecomment-127401270%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Otherwise%2C%20would%20like%20more%20comments%20%26%20more%20expressive%20fn%20names%2C%20but%20we%27re%20in%20crunch%20mode%2C%20so%20I%27m%20merging%20it%20as%20it%20is.%22%2C%20%22created_at%22%3A%20%222015-08-03T20%3A41%3A12Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/372075%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ssebro%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%202e0613f39f29eaad5303e6031043fedd32aca045%20lib/includes.js%2019%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/agco/harvesterjs/pull/126%23discussion_r36112934%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3A%28%20yoda%22%2C%20%22created_at%22%3A%20%222015-08-03T18%3A09%3A53Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/372075%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ssebro%22%7D%7D%2C%20%7B%22body%22%3A%20%22Damn%21%20It%27s%20my%20mother%20tongue%20%3A%29%20%5Cr%5CnOk%2C%20I%27ll%20rewrite%20it.%22%2C%20%22created_at%22%3A%20%222015-08-03T18%3A18%3A42Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/431064%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/blabno%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20lib/includes.js%3AL19-82%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Pull 2e0613f39f29eaad5303e6031043fedd32aca045 lib/includes.js 19'></a> &nbsp;<b><a href='https://github.com/agco/harvesterjs/pull/126#discussion_r36112934'>File: lib/includes.js:L19-82</a></b>
- <a href='https://github.com/ssebro'></a> :( yoda
- <a href='https://github.com/blabno'></a> Damn! It's my mother tongue :)
Ok, I'll rewrite it.
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/agco/harvesterjs/pull/126#issuecomment-127401270'>General Comment</a></b>
- <a href='https://github.com/ssebro'></a> Otherwise, would like more comments & more expressive fn names, but we're in crunch mode, so I'm merging it as it is.

<a href='https://www.codereviewhub.com/agco/harvesterjs/pull/126?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/agco/harvesterjs/pull/126?approve=1'></a>&nbsp;<a href='https://github.com/agco/harvesterjs/pull/126'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by ssebro** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/agco/harvesterjs/pull/126'>Refresh</a>.<a href='#crh-update-none'></a>

File: lib/includes.js [hunk 1]
```diff
@@ -20,15 +19,63 @@ module.exports = function (adapter, schemas) {
...
-            .map(function (inclusion) {
-                var inclusionTokens = inclusion.split('.');
-                return buildInclusionBranch(inclusionTokens, primarySchema);
-            })
-            .reduce(function (acc, branch) {
-                return _.merge(acc, branch);
-            })
-            .value();
+        function isRemote(branch, key) {
+            return null != branch && null != branch.refs && null != branch.refs[key] && null != branch.refs[key].def && null != branch.refs[key].def.baseUri;
```
Comment (ssebro) [inline lib/includes.js]: :( yoda

Comment (blabno) [inline lib/includes.js]: Damn! It's my mother tongue :) 
Ok, I'll rewrite it.

**Comment by ssebro** [comment]
Otherwise, would like more comments & more expressive fn names, but we're in crunch mode, so I'm merging it as it is.

**Pull request merged**